static/src/stylesheets/module/content/_article-immersive.scss

```diff
@@ -112,15 +112,10 @@
     font-weight: 200;
 }
 
-.immersive-main-media__headline-container--dark {
-    .content__headline--immersive-article {
-        color: #ffffff;
-    }
-}
-
```

Comment:
Can we just re-add this block, we'll get best of both worlds - no overriding via tone for this specific case, and a sensible default in the general case?  Don't know your case that well, so it's a bit of a guess.